docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -33,7 +33,9 @@ Webhook payloads are sent in POST requests.
 
 All webhooks with `targetUrl`, where the scheme is HTTP(S) will use this protocol.
 
-Saleor calculates the HMAC sha256 header - `Saleor-Signature` based on `secretKey` and the payload.
+Saleor calculates a payload signature under `Saleor-Signature`:
+ * if `secretKey` was set for a webhook - the HMAC sha256 header based on it and the payload
+ * if `secretKey` wasn't set for a webhook - the RS256 signature based on our private key used also to sign JTW tokens and the payload (available since Saleor 3.5, before payload from webhooks without `secretKey` set were not signed)
```

Comment:
```suggestion
 * if `secretKey` wasn't set for a webhook - the RS256 signature based on our private key used also to sign JWT tokens and the payload (available since Saleor 3.5, before payload from webhooks without `secretKey` set were not signed)
```